presto-hive/src/main/java/io/prestosql/plugin/hive/HiveMetadata.java

```diff
@@ -301,6 +307,12 @@
     private final HiveStatisticsProvider hiveStatisticsProvider;
     private final AccessControlMetadata accessControlMetadata;
 
+    // Copied from IcebergTableHandle
+    private static final Pattern ICEBERG_TABLE_NAME_PATTERN = Pattern.compile("" +
```

Comment:
Should we simplify this so that we don't depend on the exact Iceberg table name format? Something like `([^$@]+)([$@].+)?` Or maybe it's better to keep it exactly the same.

Reply:
Hmm I'd vote for keeping it exactly the same